examples/rbac-local/README.md

```diff
@@ -0,0 +1,57 @@
+# RBAC demo
+RBAC demo with local environment.
+
+## System Requirements
+* Clone of the Feast repo
+* Docker
+* yq
```

Comment:
what is `yq`

Reply:
I thought it was not worth to mention it 😮 
[a lightweight and portable command-line YAML, JSON and XML processor ](https://github.com/mikefarah/yq)

Reply:
may be better to mention it as a link. Hope i am not an ignorant if it is a popular library.
* [yq](https://github.com/mikefarah/yq)